sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -536,6 +549,18 @@ public EncryptedBlobClientBuilder pipeline(HttpPipeline httpPipeline) {
         return this;
     }
 
+    /**
+     * Sets the client options for all the requests made through the client.
+     *
+     * @param clientOptions {@link ClientOptions}.
+     * @return the updated EncryptedBlobClientBuilder object
+     * @throws NullPointerException If {@code clientOptions} is {@code null}.
+     */
+    public EncryptedBlobClientBuilder clientOptions(ClientOptions clientOptions) {
+        this.clientOptions = Objects.requireNonNull(clientOptions, "'clientOptions' cannot be null.");
```

Comment:
In other SDKs I've gone with allowing `ClientOptions` to be null with having a static default instance when null is passed.

Reply:
I think the other options on the storage builders enforce non null just like this even though we have a default instance. Should we try to be consistent within storage or with the rest of the libraries?

Reply:
@alzimmermsft Any thoughts?

Reply:
I would go consistency within Storage over consistency across other SDKs in this case.